linters/.eslintrc

```diff
@@ -1,5 +0,0 @@
-// Use this file as a starting point for your project's .eslintrc.
```

Comment:
ember-cli gives us a starting point with the non-deprecated .eslintrc.js, we don't need a file template